### ISSUE TITLE
Double API hit count for Nexrad Plot

### ISSUE THREAD

**Issue by shu3hamiitkgp** [BigDataIA-Spring2023-Team-09/Assignment_3#15]
When selected, the Nexrad plot counts twice for API calls instead of a single hit.

**Comment by Hmittal15** [comment]
Yes that is correct. This is a Streamlit behavior. When a user navigates to 'Mapping' page, the map loads up and 1 API call is registered. Streamlit then again reloads the map which marks as another call. We can configure it to count that as a single call only. Thanks for highlighting.
